dataverse-persistence/src/main/resources/db/migration/V12__addedUserPreferredLanguage.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE authenticateduser
+    add column if not exists notificationslanguage varchar(2);
+
+UPDATE authenticateduser
+    SET notificationslanguage = 'en'
+    WHERE notificationslanguage IS NULL;
```

Comment:
Do we want to have situations where `notificationslanguage` is `null`? IMO we should not. If that's the case then we should end up with column being defined as not null

Reply:
Ok. My original thought was that it shouldn't be a big of a deal if `notificationslanguage` is `null` but we might aswell make it not nullable. 
And this was just a migration code.